src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item-info/ProteinsListItemInfo.scss

```diff
@@ -60,3 +57,41 @@ $content-width: $gene_image_width;
   justify-content: center;
   margin-top: 40px;
 }
+
+.xrefWithChevron {
+  display: flex;
+}
+
+.xrefGroupChevron {
+  text-align: right;
+  padding-right: 2px;
+  cursor: pointer;
+  font-weight: $normal;
+  min-width: 50px;
+}
+
+.chevron {
+  margin-left: 6px;
+  margin-bottom: -1px;
+  height: 12px;
+  width: 12px;
+}
+
+.chevronUp {
+  transform: rotate(0deg);
```

Comment:
Isn't this redundant? It would help if you animated the chevron's transition between the up and down state (as is done e.g. in expandable sections with species stats on the species page; but if you aren't going to animate the transition, what's the point of this rule?

Reply:
This was just to make the css naming good when used inside the component